packages/job-worker/src/playout/cache.ts

```diff
@@ -124,6 +125,7 @@ export class CacheForPlayout extends CacheForPlayoutPreInit implements CacheForS
 	private toBeRemoved = false
 
 	public readonly Timeline: DbCacheWriteOptionalObject<TimelineComplete>
+	public readonly TimelineDatastore: DbCacheWriteCollection<DBTimelineDatastoreEntry>
```

Comment:
I am still currently thinking that it would make sense to not include this on the cache.
Because it is present during the action-execution (where adlib-actions could modify it), having it here feels risky.

As it only gets used during deactivation, I think it acceptable for that to be done directly on the database for now. We can always change it back once we have a clearer plan on where else it would be useful
(Idea for later: this could use a `DbCacheWriteSyncedCollection`? all modifications get immediately pushed to the database, as well as updating a cached copy. That might be useful in other places too)